docs/source/cli/xcube.rst

```diff
@@ -32,14 +32,21 @@
     
     Commands:
       chunk     (Re-)chunk xcube dataset.
+      compute   Compute a cube from one or more other cubes.
       dump      Dump contents of an input dataset.
+      edit      Edit the metadata of an xcube dataset.
       extract   Extract cube points.
       gen       Generate xcube dataset.
-      grid      Find spatial xcube dataset resolutions and adjust bounding boxes.
+      grid      Find spatial xcube dataset resolutions and adjust bounding...
```

Comment:
Apparently this truncation is a click ‘feature’ and can be worked around by providing the description as a `short_help` parameter to `@click.group` rather than letting click extract it automatically from the docstring (see https://github.com/pallets/click/issues/486 ).

Reply:
Thanks, but I feel this is minor.